### PR TITLE
Add a group

### PR DESCRIPTION
**Pull request by JelenadlS**

### CODE REVIEW

File: src/pages/AddFriendPage.js [hunk 1]
```diff
@@ -72,7 +83,10 @@ export default function AddFriendPage({
             </AddButton>
             {tooLong === true && (
               <StyledNotification>
-                <i>This is quite a long name for a friend, make it shorter</i>
+                <i>
+                  This are quite a lot friends keep the overview and make it
```
Comment (ttaannjjaa) [inline src/pages/AddFriendPage.js:87]: "These" ?   instead of "This"   , a lot of friends?

File: src/pages/AddGroupPage.js [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+    setEnteredGroup(event.target.value);
+  }
+
+  function onDeleteGroup(thisGroupId) {
+    setAddedGroup(addedGroup.filter(group => group.id !== thisGroupId));
+  }
+}
+
+const Grid = styled.span`
+  height: 85vh;
```
Comment (ttaannjjaa) [inline src/pages/AddGroupPage.js:141]: please check, whether you wanted to use here `span`element as a Grid

**Review by ttaannjjaa — APPROVED** [review]
A lot of work - great job 👍 only 3 comments to check

File: src/pages/MyFriendsPage.js [hunk 3]
```diff
@@ -36,17 +42,18 @@ export default function MyFriendsPage({
     if (firstFriend > secondFriend) return 1;
     return 0;
   });
-
+  console.log(friendsOnlyOnceWithoutDetails);
```
Comment (ttaannjjaa) [inline src/pages/MyFriendsPage.js]: please check, whether this is indeed removed

File: src/pages/MyFriendsPage.js [hunk 4]
```diff
@@ -36,17 +42,18 @@ export default function MyFriendsPage({
...
       <Main>
-        {activities.length > 0 ? (
+        {activitiesWithoutFriend.length > 0 ||
+        friendsOnlyOnceWithoutDetails.length > 0 ? (
           <>
             {activitiesWithoutFriend.length > 0 && (
               <StyledLink to="/I still need to plan...">
                 <NameStyling>
-                  Things I still need to plan with someone:
+                  Aktivities I still need to plan with someone:
```
Comment (ttaannjjaa) [inline src/pages/MyFriendsPage.js:56]: i think here would be "Activities" the correct spelling

File: src/pages/MyFriendsPage.js [hunk 5]
```diff
@@ -74,8 +81,75 @@ export default function MyFriendsPage({
           </>
         ) : (
           <StyledEmptyMessage data-testid="emptylist">
-            Unfortunately you did not enter any activity yet. Start now and fill
-            your list with amazing activities!
+            <StyledIntro>
+              Hi there! 👋 <br />
+              There are no activities entered yet.
```
Comment (ttaannjjaa) [inline src/pages/MyFriendsPage.js:86]: maybe replace "entered" with "there"

File: src/pages/MyFriendsPage.js [hunk 6]
```diff
@@ -74,8 +81,75 @@ export default function MyFriendsPage({
...
+            </p>
+
+            <MainNavButtonSmall>
+              <img width="40" height="27" src={newIcon} alt="new" />
+            </MainNavButtonSmall>
+            <p>
+              <strong>Add an activity!</strong> <br />
+              Are you done and added a friend? <br />
+              Click on this button in the navigation and add an acticity!
+            </p>
```
Comment (ttaannjjaa) [inline src/pages/MyFriendsPage.js:128]: typo   activity

File: src/pages/MyFriendsPage.js [hunk 7]
```diff
@@ -74,8 +81,75 @@ export default function MyFriendsPage({
...
+              height="30"
+              alt="friendsHomeIcon"
+              src={inspireIcon}
+            />
+            <p>
+              <strong>Get inspired!</strong>
+              <br />
+              As soon as you did an activity, you can mark it as liked or not
+              liked in the overview. Whenever you are looking for an amazing
+              activity you can be inspred by your old ones you liked.
```
Comment (ttaannjjaa) [inline src/pages/MyFriendsPage.js:151]: inspired?

File: src/pages/MyGroupsPage.js [hunk 8]
```diff
@@ -37,22 +43,97 @@ export default function MyGroupsPage({
...
+              <br />
+              As soon as you add activities, a list with an overview of your
+              friends you planned activities with will appear.
+            </p>
+            <img width="40" height="20" alt="friendsHomeIcon" src={groupIcon} />
+            <p>
+              <strong>Activities planned with a group</strong> <br />
+              Here you can find a list of all groups with their individual
+              activities you entered.
+            </p>
```
Comment (ttaannjjaa) [inline src/pages/MyGroupsPage.js:87]: "added"   instead of  "entered" maybe?

File: src/pages/MyGroupsPage.js [hunk 9]
```diff
@@ -37,22 +43,97 @@ export default function MyGroupsPage({
...
+
+            <MainNavButtonSmall>
+              <img width="40" height="27" src={newIcon} alt="new" />
+            </MainNavButtonSmall>
+            <p>
+              <strong>Add an activity!</strong> <br />
+              Are you done and added a friend and or a group? <br />
+              Click on this button in the navigation and add an acticity!
+            </p>
```
Comment (ttaannjjaa) [inline src/pages/MyGroupsPage.js:108]: activity

**Review by ttaannjjaa — APPROVED** [review]
your code looks good to me, great work, just some typos

File: src/pages/MyFriendsPage.js (hunk 3 above)
Comment (JelenadlS) [reply src/pages/MyFriendsPage.js]: its already out -  thank you! 😊

**Pull request merged**